recipes/prefixed/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "prefixed" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ca48277ba5fa8346dd4b760847da930c7b84416387c39e93affef086add2c029
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=2.7,!=3.0,!=3.1,!=3.2,!=3.3,!=3.4,<4
```

Comment:
While this is correct, in conda-forge we don't build anything <=3.5. So this can be simplified to `>=3.6,<4`. I'll leave it to your taste to change it or not in the future. Merging this...